openhub/management/commands/import_organization_data.py

```diff
@@ -0,0 +1,12 @@
+from django.core.management.base import BaseCommand
+from openhub.organization import get_organization_data, import_data
+
+
+class Command(BaseCommand):
+    help = 'Import Organization data'
+
+    org = get_organization_data()
+
+    def handle(self, *args, **options):
+
```

Comment:
remove

Reply:
Actually, this is the only command which doesn't have identical `handle` with other.

Reply:
remove the blank line